src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,37 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const userStr = userInput.toString();
+  const guessStr = numberToGuess.toString();
+
+  let bulls = 0;
+  let cows = 0;
+  const guessDigitCount = {};
+  const userDigitCount = {};
+
+  for (let i = 0; i < 4; i++) {
+    if (userStr[i] === guessStr[i]) {
+      bulls++;
+    } else {
+      if (!guessDigitCount[guessStr[i]]) {
+        guessDigitCount[guessStr[i]] = 0;
+      }
+
+      if (!userDigitCount[userStr[i]]) {
+        userDigitCount[userStr[i]] = 0;
+      }
+      guessDigitCount[guessStr[i]]++;
+      userDigitCount[userStr[i]]++;
+    }
+  }
+
+  for (const digit in userDigitCount) {
```

Comment:
This works, but seems overcomplicated...
Instead of storing values to objects and them iterating, you could use only a single _for loop_ to solve, why don't you give it a try?